krux_sqs/sqs.py

```diff
@@ -84,6 +84,11 @@ def add_sqs_cli_arguments(parser, include_boto_arguments=True):
     group = get_group(parser, NAME)
 
 
+def sqs_send_message(queue_name, message):
+    """Utility function for sending SQS messages."""
+    Sqs(Boto3()).send_message(queue_name, message)
```

Comment:
There are number of problems with this approach.

1. This provides no way to override the authentication method for Boto. `krux-boto` object will always use default authentication method (env var).
1. This nullifies the built in caching of SQS queue objects in `krux-sqs`. With `Sqs` object, the queue will be cached and reused. This approach will re-instantiate everything.